js/binance.js

```diff
@@ -1376,8 +1393,8 @@ module.exports = class binance extends Exchange {
             };
         }
         let takerOrMaker = undefined;
-        if ('isMaker' in trade) {
-            takerOrMaker = trade['isMaker'] ? 'maker' : 'taker';
+        if ('maker' in trade) {
```

Comment:
We may be breaking the spot trades if we change this line this way ↑

https://github.com/ccxt/ccxt/pull/8417/files#diff-13b40969472b078d4e2cb520b264e56efb07c603587c854842cfc45c946f7b82R1329

I think we should support both fields, what do you say?

Reply:
oh, yes of course

Reply:
er, I can't figure out how to edit this pr!

Reply:
@npomfret no problem, i'll edit it and will merge shortly today. Thank you!